tools/data-api/internal/repositories/services_test.go

```diff
@@ -0,0 +1,27 @@
+package repositories
+
+import (
+	"testing"
+)
+
+func TestServices_ResourceManager(t *testing.T) {
+	repo, err := NewServicesRepository("../../../../api-definitions/", ResourceManagerServiceType, nil)
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	if _, err := repo.GetAll(ResourceManagerServiceType); err != nil {
+		t.Fatalf(err.Error())
+	}
+}
+
+func TestServices_Graph(t *testing.T) {
```

Comment:
To be pedantic, can we keep this consistent as `MicrosoftGraph`? There's different kinds of `Graph` referenced in Azure - (AAD Graph, MS Graph, Resource Graph) so since we're being consistent elsewhere it'd be nice to be consistent here too:

```suggestion
func TestServices_MicrosoftGraph(t *testing.T) {
```

Reply:
Good shout! Ty ty